index.html

```diff
@@ -9,8 +9,11 @@
   <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@300;400;500&display=swap" rel="stylesheet">
   <title>Mobile first portfolio project</title>
   <link rel="stylesheet" href="styles.css">
+  <link rel="stylesheet" href="https://kit.fontawesome.com/a13da07216.css" crossorigin="anonymous">
+  <script src="https://kit.fontawesome.com/a13da07216.js" crossorigin="anonymous"></script>
 </head>
 <body>
```

Comment:
- [ ] One of the main requirements of this whole project is to create a responsive design at all screen sizes from `375px` to above. the problem is that you are giving your body an exact width of `375px` at line 8 in `style.css` file. 
   -  we actually avoid giving exact widths to our  containers because it makes almost impossible to achieve responsiveness. instead we try to give widths in percentages `%` and in `vw`. so kindly try to make your design responsive from `375px` to `768px`.
  your design
![qw](https://user-images.githubusercontent.com/96683843/224437002-734ea973-2ab5-4ef8-85de-4518206aa180.jpg)
![as](https://user-images.githubusercontent.com/96683843/224437019-89873126-35dd-4e93-94a3-c362251c74d1.jpg)